gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/CoopLockFsckRunner.java

```diff
@@ -114,6 +121,16 @@ public int run() throws IOException {
       return 0;
```

Comment:
Move this to start of the method.

Reply:
In this case it will not print operations status, which is a purpose of the check command.